README.md

```diff
@@ -323,15 +321,15 @@ exec('some_long_running_process', function(code, stdout, stderr) {
 ```
 
 Executes the given `command` _synchronously_, unless otherwise specified.  When in synchronous
-mode, this returns a ShellString (compatible with ShellJS v0.6.x, which returns an object
+mode, this returns a `ShellString` (compatible with ShellJS v0.6.x, which returns an object
 of the form `{ code:..., stdout:... , stderr:... }`). Otherwise, this returns the child process
-object, and the `callback` gets the arguments `(code, stdout, stderr)`.
+object, and the `callback` receives the arguments `(code, stdout, stderr)`.
 
 Not seeing the behavior you want? `exec()` runs everything through `sh`
 by default (or `cmd.exe` on Windows), which differs from `bash`. If you
 need bash-specific behavior, try out the `{shell: 'path/to/bash'}` option.
 
-**Note:** For long-lived processes, it's best to run `exec()` asynchronously as
+**Note:** For long-lived processes, it's best to run `exec()` asynchronously, as
```

Comment:
Actually, you can just remove this. The CPU concerns are no longer relevant.

Reply:
Will do!